examples/kubernetes/1.10/cilium.yaml

```diff
@@ -391,6 +402,24 @@ spec:
                 key: cluster-id
                 name: cilium-config
                 optional: true
+          - name: AWS_ACCESS_KEY_ID
```

Comment:
Instead of running the AWS secrets as an environment variable can we mount it as a volume and read the file created from there?

Reply:
The way that AWS usually work is like this, so we can read the data from the file, but using this way will be easy to understand for AWS users.

Reply:
>The way that AWS usually work is like this, so we can read the data from the file
What file?